src/app/client/src/app/modules/groups/components/add-member/add-member.component.html

```diff
@@ -1,45 +1,45 @@
 <!-- add add-member screen -->
 
-<div>
-  <!-- Add member with profile images content -->
-  <!-- <app-create-group></app-create-group> -->
+<div *ngIf="memberCreationStep == 1">
+  <div class="flex-dc d-flex flex-ai-center flex-jc-center create-group-container">
+    <div class="d-flex flex-dc flex-ai-center flex-jc-center p-24">
+     <div class="d-flex flex-as-center"><img src="./assets/images/create-group-img.png" width="220px" height="220px"></div>
+     <div class="my-24 fs-1 text-center">{{ resourceService.frmelmnts.lbl.createGroupMsg }}</div>
+       <button class="sb-btn sb-btn-primary sb-btn-md flex-ai-jc-center mb-8" id="addMemberStep1" type="button" (click)="memberCreationStep = 2">
+        <img src="./assets/images/icon_group.svg" width="24px" height="24px" class="mr-4">
+         {{ resourceService.frmelmnts.lbl.addMember }}
+       </button>
+    </div>
+  </div>
 </div>
 
 <!-- Add member with verify button content -->
-<div>
+<div *ngIf="memberCreationStep > 1">
   <div class="title my-16 font-weight-bold">
-    Add Member
+    {{ resourceService.frmelmnts.lbl.userId }}
   </div>
   <div class="sb-field">
-    <label for="id">User ID</label>
-    <input type="id" name="id" class="sb-form-control is-invalid" placeholder="Enter user ID">
-    <small class="message sb-color-error font-weight-bold">Invalid user ID</small>
+    <label for="id">{{ resourceService.frmelmnts.lbl.addMember }}</label>
+    <input type="text" name="userid" id="userid" [value]="userid" (input)="userid = $event.target.value;" class="sb-form-control" placeholder="Enter user ID" [ngClass]="{'is-invalid': invalidUserid}">
+    <small class="message sb-color-error font-weight-bold" *ngIf="invalidUserid">{{ resourceService.frmelmnts.lbl.invalidUserid }}</small>
   </div>
-  <button type="button" class="sb-btn sb-btn-primary sb-btn-normal sb-btn-md mt-16 w-100">
-    Verify
+  <button type="button" class="sb-btn sb-btn-primary sb-btn-normal sb-btn-md mt-16 w-100" *ngIf="memberCreationStep == 2" (click)="VerifyMemberUserId()">
+    {{ resourceService.frmelmnts.btn.verify }}
   </button>
-</div>
 
-<!-- Add to group button content -->
-<div>
-  <div class="title my-16 font-weight-bold">
-    Add new Member
-  </div>
-  <div class="sb-field">
-    <label for="id">User ID</label>
-    <input type="id" name="id" class="sb-form-control is-invalid" placeholder="Enter user ID">
-    <small class="message sb-color-error font-weight-bold hide">Invalid user ID</small>
-  </div>
-  <div class="d-flex flex-jc-center flex-ai-center add-member-content py-24">
-    <div class="add-member-text text-center">
-      <div class="oval">A</div>
-      <div class="fs-1 mt-4 font-weight-bold">Gupta</div>
+  <!-- Add to group button content -->  
+  <div *ngIf="memberCreationStep == 3" >
```

Comment:
instead of steps use the enum with the current state of the group creation

Reply:
resolved